### PR TITLE
T519b

### PR DESCRIPTION
**Pull request by dseif**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 0

Live output at: http://butyr.org:8000/485137dbb61e572/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 1

Live output at: http://butyr.org:8000/dd4e9a5b13fd80a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/485137dbb61e572/output.txt

Total script time: 0.22 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/dd4e9a5b13fd80a/output.txt

Total script time: 0.57 mins

Server is running on http://pr299.butyr.org/templates/test.html

File: src/timeline/media.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ define( [
 
       _currentMouseDownTrackEvent = trackEvent;
 
-      if( trackEvent.selected === true && originalEvent.shiftKey && _selectedTracks.length > 1 ){
+      if( trackEvent.selected === true && originalEvent.shiftKey && _selectedTracks.length > 1  && originalEvent.srcElement.className.split( " " ).indexOf( "handle" ) === -1 ){
```
Comment (secretrobotron) [inline src/timeline/media.js]: Can you just split up this line a bit? It's huge now!

Comment (dseif) [inline src/timeline/media.js]: Yeap will do

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 0

Live output at: http://butyr.org:8000/085d0b2a7f876b8/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/085d0b2a7f876b8/output.txt

Total script time: 0.24 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://butyr.org:8000/2014d7f5ed7076c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/2014d7f5ed7076c/output.txt

Total script time: 0.40 mins

Server is running on http://pr299.butyr.org/templates/test.html

File: src/timeline/media.js [hunk 2]
```diff
@@ -22,7 +23,8 @@ define( [
           TimeBar,
           ZoomBar,
           Status,
-          TrackHandles ){
+          TrackHandles,
+          LangUtil ) {
```
Comment (secretrobotron) [inline src/timeline/media.js:28]: Can you just add a comment here about why you're including this?

Comment (secretrobotron) [inline src/timeline/media.js:28]: It's just non-obvious for people who don't know there's a shim hiding in there.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/6fe0d864ad1c6a9/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/6fe0d864ad1c6a9/output.txt

Total script time: 0.32 mins

**Pull request merged**